### DISCUSSION TITLE
Valgrind + funcheck

### DISCUSSION THREAD

**Discussion by steffenparth** [froz42/funcheck#106] [Q&A] [answered]
Do you know if there is a way to run Valgrind and Funcheck at the same time? When I try it, the program always gets stuck. I think it would be interesting to see if there are memory leaks when a function fails.

**Comment by froz42** [comment, marked as answer]
Hello, you can run funcheck like so
```bash
funcheck -a ./your_prog
```
equivalent to:
```bash
funcheck --track-allocations ./your_prog
```
You can also check help section
```bash
funcheck -h
```
kind regards

**Reply by steffenparth** [reply]
Great - thank you !
